### PR TITLE
add More engine

### PR DESCRIPTION
**Pull request by MythHack**
googleter dnsdb hackertarget

### CODE REVIEW

File: engines/engine.py [hunk 1]
```diff
@@ -614,6 +614,147 @@ def extract_domains(self, resp):
...
+                        self.print_("%s%s: %s%s" % (self.logger.R, self.engine_name, self.logger.W, subdomain))
+                    self.subdomains.append(subdomain.strip())
+        except Exception as e:
+            pass
+
+
+class DnsDB(EnumratorBaseThreaded):
+    def __init__(self, domain, subdomains=None, q=None, silent=False, logger=None):
+        subdomains = subdomains or []
+        base_url = 'http://www.dnsdb.org/f/{domain}.dnsdb.org/'
```
Comment (Plazmaz) [inline engines/engine.py]: Please use SSL whenever possible.

File: engines/engine.py [hunk 2]
```diff
@@ -614,6 +614,147 @@ def extract_domains(self, resp):
...
+            return self.subdomains
+
+        self.extract_domains(resp)
+        return self.subdomains
+
+    def extract_domains(self, resp):
+        try:
+            subdomains = re.findall(r"(?<=href=\").+?(?=\")|(?<=href=\').+?(?=\')", resp)
+            for subdomain in subdomains:
+                subdomain = subdomain.replace('http://', '').replace('.dnsdb.org/', '')
```
Comment (Plazmaz) [inline engines/engine.py]: Will also need to be changed here.

**Comment by MythHack** [comment]
fix

**Comment by Plazmaz** [comment]
Looks good! Thanks!

**Pull request merged**